shell.nix

```
@@ -0,0 +1,45 @@
+{ pkgs ? import <nixpkgs> {}
+, toLower ? pkgs.lib.toLower
+, python ? pkgs.python39
+, pythonPackages ? pkgs.python39Packages
```

```suggestion
, python ? pkgs.python36
, pythonPackages ? pkgs.python36Packages
```
Since the README says we should use python 3.6, we should do the same in the shell